apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -60,4 +67,17 @@ export class FilesStorageController {
 
 		return response;
 	}
+
+	@Patch(':id/rename')
+	async patchFilename(
+		@Param('id', ParseObjectIdPipe) fileRecordId: string,
+		@Body() params: RenameParams,
```

Comment:
Params or Body? For same handling for the mongooseId i prefare that we have somthing like this
export class SingleFileParams{
	@ApiProperty()
	@IsMongoId()
	fileRecordId!: EntityId;
}

@Patch(':rename/fileRecordId')
@Param() params: SingleFileParams,